apps/sms/test/marionette/drafts_test.js

```diff
@@ -283,6 +289,34 @@ marionette('Messages Drafts', function() {
       });
     });
 
+    suite('Thread-less recipient-less drafts', function() {
```

Comment:
I think they are the same, also in inbox-view terms conversation-less draft is "conversation" as well, @julienw please correct me if I'm wrong :)